### PR TITLE
XRDDEV-227: change Security Server's address from Security Server UI

### PR DESCRIPTION
**Pull request by justasnortal**
Refs: XRDDEV-227

### CODE REVIEW

**Comment by justasnortal** [comment]
[rerun-ci]

File: src/security-server/system-test/src/intTest/resources/behavior/01-ui/0400-ss-system-parameters.feature
Comment (andresrosenthal) [inline src/security-server/system-test/src/intTest/resources/behavior/01-ui/0400-ss-system-parameters.feature:1]: Ideally the success scenario of Security server address change should also be added.
I assume it was left out due to requiring quite a lot of extra effort:
* Mocking the address change management service
* Mocking updated globalconf with the new address

As this PR is quite huge already I guess it's okay to leave it out for now, but should be added in the future.

**Review by andresrosenthal — APPROVED** [review]
Looks good to me

**Comment by justasnortal** [comment]
[rerun-ci]

**Pull request merged**